net-mail/dbmail/dbmail-3.2.5.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd
+
+DESCRIPTION="Fast and scalable sql based email services"
+HOMEPAGE="https://www.dbmail.org/"
+SRC_URI="https://github.com/dbmail/dbmail/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+doc jemalloc ldap libressl sieve ssl static systemd"
+
+DEPEND="dev-db/libzdb
+	sieve? ( >=mail-filter/libsieve-2.2.1 )
+	ldap? ( >=net-nds/openldap-2.3.33 )
+	jemalloc? ( dev-libs/jemalloc )
+	app-text/asciidoc
+	app-text/xmlto
+	app-crypt/mhash
+	sys-libs/zlib
+	dev-libs/gmime:2.6
+	>=dev-libs/glib-2.16
+	dev-libs/libevent
+	ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)"
+RDEPEND="${DEPEND}
+	acct-group/dbmail
+	acct-user/dbmail"
+DOCS=( AUTHORS README.md INSTALL THANKS UPGRADING )
+
+src_configure() {
+	econf \
+		--enable-manpages \
+		--sysconfdir=/etc/dbmail \
+		$(use_enable doc manpages) \
+		$(use_enable static) \
+		$(use_enable systemd) \
+		$(use_with jemalloc) \
+		$(use_with sieve) \
+		$(use_with ldap auth-ldap)
+}
+
+src_install() {
+	emake DESTDIR="${D}" SYSTEMD_UNIT_DIR="$(systemd_get_systemunitdir)" install
+	einstalldocs
+
+	docompress -x /usr/share/doc/${PF}/sql
+	dodoc -r sql
+	dodoc -r test-scripts
+	dodoc -r contrib
+	## TODO: install other contrib stuff
+
+	sed -i -e "s:nobody:dbmail:" dbmail.conf || die
+	sed -i -e "s:nogroup:dbmail:" dbmail.conf || die
+	sed -i -e "s:/var/run:/run/dbmail:" dbmail.conf || die
```

Comment:
Please combine these to a single `sed` call since they edit the same file.

Can this operation be moved to `src_prepare` where it belongs?